### ISSUE TITLE
Upload Data Only

### ISSUE THREAD

**Issue by GerhardSc** [VFPX/UpsizingWizard#10]
Hello Doug,

I would like to upload data only to an existing SQL Database. Is there any way to do this?

The reason I want this is, that I have lots of constraints that I have a *.sql to create the database with that very constraints. But when upsizing the structure the constraints are changed. This is especially the case for NULL / NOT NULL. It would be too much work to change the settings in the upsizing wizard, every time I run it. 
I searched the code to skip the database creation but did not find it.

TIA
Gerhard

**Comment by DougHennig** [comment]
I think you mean not creating the tables, correct? If you really mean not creating the database, then if you are using the Upsizing Wizard UI, in step 3, choose Existing and select the database to upsize to. If you're calling it programmatically instead, set the CreateNewDB property to .F.

If you mean not creating the tables, then you'll have to using the Upsizing Wizard programmatically and either use an extension object with a CreateTables method that returns .F. (meaning don't create tables) or changing the CreateTables method in WizUsz.prg to do nothing.

Doug

**Comment by GerhardSc** [comment]
Thank you for your quick response.
Yes, I want to avoid creating the tables. I will try to change the CreateTables to do nothing.

Gerhard

**Comment by GerhardSc** [comment]
I tried to change the Upsizing Wizard to prevent it from creating the table but was not successful. Therefore I used another approach now: I use the Upsizing Wizard to create the database and the table and to load the data. After that, I run a *.sql to adjust the constraints as I need them.
Sorry for taking your time, but I am under pressure to deliver and therefore had to get the job done.
Thanks again for your support!
Gerhard
